apps/foundation-demo/src/app/product/components/product/product.component.ts

```diff
@@ -11,12 +11,17 @@ import { Image } from '../../../design/interfaces/image';
 })
 export class ProductComponent implements OnInit {
   @Input() product: Product;
-
+  @Output() addToCart: EventEmitter<any> = new EventEmitter();
+  
+  qty: number;
```

Comment:
You're storing a copy of local state here. Can we store this in redux?

Reply:
done